### ISSUE TITLE
The bottom bar that is "position: fixed" interferes on /print page

### ISSUE THREAD

**Issue by thadk** [codeforboston/windfall-elimination#153]
The bottom bar blocks the printout for the print page. 

Please either: 
* add a special print media CSS rule where the bar is not `position: fixed`
* or have that page use a different style for its bottom bar on that page.

<img width="1024" alt="image" src="https://user-images.githubusercontent.com/283343/72634796-532b2400-3929-11ea-9eae-7d2c6eea16f8.png">

**Comment by riiniii** [comment]
can I do this one? / may I help out, even though I am not in boston?

**Comment by thadk** [comment]
@riiniii Welcome to submit the PR and I'll review it. Thanks for the interest.

**Comment by riiniii** [comment]
@thadk sorry for the lateness. does this issue still persist? tried to reproduce, i got a different result. are you using an ipad? 
![image](https://user-images.githubusercontent.com/45681867/75355319-4666fc00-58e9-11ea-842f-2f94f73d27b1.png)

**Comment by thadk** [comment]
Hi @riiniii – looked into this again and the issue is still happening on the iPad but it does not happen on Safari so it might be hard to troubleshoot without an iPad.
I also reviewed the code that is used when "Print Results" button is pressed and it is not standard React code, but directly manipulates the DOM. I suspect this non-standard code is not working on iPad: https://github.com/codeforboston/windfall-elimination/blob/79d26d12648737c60c3180471ff882dc83391ac8/src/pages/print.tsx#L192-L202

Overall, the correct solution is to drop this code and use a print media query for the printout. Since we are using CSS-in-JS called [Emotion](https://github.com/emotion-js/emotion), the change might not just be adding a stylesheet. I'm not sure yet why the developer who contributed this wasn't able to take that approach.

We're looking into how to do several media-queries well for the app to support #155 and I just noticed https://github.com/emotion-js/facepaint . Maybe this is the way to solve this issue. It is probably not a good first issue any more though unless you know your way around CSS-in-JS comfortably.
